### DISCUSSION TITLE
Making daily challenge engaging for all skill levels

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#29087] [Ideas]
Daily challenge is pretty fun, but there is nothing preventing higher skilled player from dominating a daily challenge with a lower diff, intended for lower skilled players. 
There are a lot of ideas thrown around to rework the mode and fix this issue, but they would often require a complete rethinking and rewrite.
How about a simpler way leaning into the existing system?

Let's imagine the leaderboard is divided into categories, let's say leagues for example to make it exciting, based on the score achieved on the map.
For example 1mil and up would be diamond league, then gold for 1mil to 800k, and so on. (Or maybe extra insane... like the difficulty names)
The user could then filter by league and get different leaderboards based on each league (maybe would default to filtering the leaderboard to the league achieved by the user).
The highest league reached could be displayed in the profile tooltip as a way to track progress (or maybe medals)

I think it's a good way to drive engagement for players of all skill level without having to rethink everything, and without compromising the original idea.

I hope it's a feasible idea and thank you for reading!

**Comment by marioman63** [comment]
I'd like to see a couple different star ranges features in each challenge, also maybe fewer free mods and more preset mods (it is a daily challenge after all, make it feel special)

**Comment by peppy** [comment]
That doesn't sound like it will scale well to all beatmaps. It's basically just splitting the leaderboard up arbitrarily no?

**Reply by vatei** [reply]
It's pretty much arbitrary numbers to demonstrate the idea, the ranges should be tuned of course.
I also thought of splitting based on percentages but it would definitely feel wonkier since daily challenge is a live event (or the user would have to wait until it's complete to get results which doesn't sound very exciting)

**Reply by vatei** [reply]
Maybe higher difficulties should always have freemods or at least always allow some mods like HT?
Should even out a bit the split leaderboards, and even without my idea I feel like it would give a chance to all players to put a score up, even if it's not a good one

**Comment by ziv-vy** [comment]
One word: deranking.

It would also be very confusing. Leagues for daily challange??
